src/patternfly/demos/Card/card-content-1.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-right"}}
```

Comment:
`pf-m-align-right` isn't doing anything. Also I would add `pf-m-align-center` to vertically align the row.

Reply:
awesome thanks!